lib/system/environ/uninstall.go

```diff
@@ -141,6 +157,12 @@ func DisableAgentServices(logger log.FieldLogger) error {
 	return trace.NewAggregate(errors...)
 }
 
+func uninstallAgentServices(svm systemservice.ServiceManager) error {
+	return uninstallServices(svm,
+		defaults.GravityRPCInstallerServiceName,
+		defaults.GravityRPCAgentServiceName)
```

Comment:
There's also [gravity-resume.service](https://github.com/gravitational/gravity/blob/7.0.16/tool/gravity/cli/clusterupdate.go#L246) which we might want to move to defaults and add here as well...